corecomponent/androidcomponent/src/main/res/navigation/navigation.xml

```diff
@@ -203,8 +209,13 @@
     <fragment
         android:id="@+id/floormap"
         android:name="io.github.droidkaigi.confsched2020.floormap.ui.FloorMapFragment"
-        android:label="@string/map_label"
-        />
+        android:label="@string/map_label">
+        <argument
+            android:name="roomId"
+            android:defaultValue="-1"
+            app:argType="integer"
+            />
+    </fragment>
```

Comment:
I thought about replacing roomId with Value Object, but there is not much benefit from making roomId a Value Object. So I defined roomId as a primitive type 🤔